nrich-validation/src/main/java/net/croz/nrich/validation/constraint/validator/BaseNullableCheckValidator.java

```diff
@@ -50,6 +60,13 @@ protected boolean isValid(Object value, Class<? extends Predicate<?>> conditionC
         return isPropertyValueValid(propertyValue);
     }
 
+    @SneakyThrows
```

Comment:
Could you take a look at Codecov comments, is it just false positive?

Reply:
We should be okay here. It's probably because of Groovy. We have tests that verify that an exception is thrown when Groovy is not present and these tests pass properly.